pkg/perf/cache.go

```diff
@@ -0,0 +1,103 @@
+package perf
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/garyburd/redigo/redis"
+	"github.com/pkg/errors"
+)
+
+const (
+	moduleName = "perf"
+)
+
+// TaskResult the result test schema
+type TaskResult struct {
+	TaskName  string      `json:"task_name"`
```

Comment:
The type is already name TaskResult, so I can perfectly call this `Name`
```suggestion
	Name  string      `json:"name"`
```